development.php

```diff
@@ -121,14 +121,6 @@
           <div class="clearfix"></div>
```

Comment:
Removed the div, but kept the BRs to have some vertical space to separate from Twitter section.